src/plcc/parse_spec/parse_syntactic_spec/parse_syntactic_spec_test.py

```diff
@@ -0,0 +1,378 @@
+from pytest import raises, mark, fixture
+from .parse_syntactic_spec import parse_syntactic_spec
+from .structs import (
+    CapturingTerminal,
+    RepeatingSyntacticRule,
+    StandardSyntacticRule,
+    LhsNonTerminal,
+    RhsNonTerminal,
+    Terminal,
+    MalformedBNFError,
+)
+from plcc.load_spec.load_rough_spec.parse_lines import Line
+from plcc.load_spec.load_rough_spec.parse_dividers import Divider
+
+
+def test_None_yields_nothing():
+    assert list(parse_syntactic_spec(None)) == []
+
+
+def test_empty_yields_nothing():
+    assert list(parse_syntactic_spec([])) == []
+
+
+def test_divider_yields_nothing():
+    lines = [makeDivider()]
+    assert list(parse_syntactic_spec(lines)) == []
+
+
+def test_blank_rule():
+    blank_rule = makeLine("<noun> ::=")
+    lines = [makeDivider(), blank_rule]
+    expected = [
+        StandardSyntacticRule(blank_rule, LhsNonTerminal("noun"), [])
```

Comment:
Splitting up the objects such as the Line and Divider into helper functions is great to reduce dependencies. I would recommend doing the same with these objects and any other objects that are seen below, this can be as simple as a helper function to create a LhsNonTerminal in this instance, the goal is to have it so if there is a change in how this is implemented, it will only need to be changed in one place as opposed to having to edit all of your tests currently. This can also be done with the syntactic rule, RhsNonTerminal, and Terminal objects. Other than that, I see no issues with your tests beyond this.